### DISCUSSION TITLE
Mounting object storage buckets as file system directories in containers

### DISCUSSION THREAD

**Discussion by Wei651** [ChameleonCloud/chi-in-a-box#268] [General]
Hi. I tried to install the cc-cloudfuse in my local container instance (Deployed using OpenStack Zun), trying to mount the object storage buckets as file systems, just like what I can do in the Chameleon instance but in container instances. The installation of cc-cloudfuse was successful. But even if I have already provided the correct user credentials, the command still fails and returns a "Failed to authenticate" message. Did I miss something to configure?

Here are the screenshots:
[Issue](https://files.slack.com/files-tmb/T97ST07MK-F05UUK38CDC-5b339f3db5/image_720.png)

[.cloudfuse configuration file](https://files.slack.com/files-tmb/T97ST07MK-F05UUK4MZ5L-5d9a9d23ee/image_720.png)

[All the files I have in the /cc-cloudfuse directory](https://files.slack.com/files-tmb/T97ST07MK-F05UYAXMTL5-4f9ba4600a/image_720.png)

**Comment by msherman64** [comment]
with those credentials, are you able to run `openstack token issue`, or otherwise authenticate to the cluster?
